### DISCUSSION TITLE
How to run electron in codespace?

### DISCUSSION THREAD

**Discussion by WBCode3** [community/community#9920] [Codespaces]
I'm want run electron in codespace or setting to --no-sandbox to google-chrome

**Comment by Danzo7** [comment]
I don't think its possible

**Reply by ghost** [reply]
Hi Danzo7, thank you for feedback.

**Comment by sethsamuel** [comment]
You might try https://github.com/devcontainers/features/tree/main/src/desktop-lite

**Comment by sourdragon** [comment]
For me `electron react boilerplate` works , when i visit the localhost:1212. It doesnt open up a separate window as it is supposed to. However the html loads as intented with hot reloading.
